core/trino-main/src/main/java/io/trino/operator/GeometryAccelerationDegree.java

```diff
@@ -0,0 +1,35 @@
+package io.trino.operator;/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import com.esri.core.geometry.Geometry;
+
+public enum GeometryAccelerationDegree
```

Comment:
Do we need this enum or can we directly use `GeometryAccelerationDegree` ?

Reply:
`com.esri.core.geometry.Geometry. GeometryAccelerationDegree` has enum name like `enumMild`. Is it okay if user provides value like `enumMild`, `enumMedium`, `enumHot`?

Our custom enum requires user to provide `MID`, `MEDIUM`, `HOT`